src/SystemCommands-VariableCommands/SycMergeVariableCommand.class.st

```diff
@@ -12,6 +12,20 @@ SycMergeVariableCommand class >> defaultMenuIconName [
 	^ #merge
 ]
 
+{ #category : #activation }
+SycMergeVariableCommand class >> fullBrowserShortcutActivation [
+	
```

Comment:
is it normal to get such empty methods?

Reply:
I handled it like this because it inherited from a class that had its own activation, however if it is incorrect I can modify the hierarchy of the classes